DeviceAdministration/Web/Scripts/Views/Dashboard/TelemetryHistory.js

```diff
@@ -135,7 +139,7 @@
             var width;
 
             pluginService = powerbi.visuals.visualPluginFactory.create();
-            singleVisualHostServices = powerbi.visuals.singleVisualHostServices;
+            singleVisualHostServices = powerbi.visuals.defaultVisualHostServices;
```

Comment:
Upon inspection of the `powerbi.visuals` object in a web browser, there is no `singleVisualHostServices` property, but there is a `defaultVisualHostServices` one. Changing the property fixes a `this.hostServices` is undefined error in the browser that prevents the visualization from loading.